component/main.jsonnet

```
@@ -0,0 +1,16 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local prom = import 'lib/prom.libsonnet';
+local rl = import 'lib/resource-locker.libjsonnet';
```

Out of all these imports, the only one you're using is `kap`, and even that one is not really required since you don't use the parameters in this file at all.

```suggestion
local kap = import 'lib/kapitan.libjsonnet';
```